3.5/SSO_README.md

```diff
@@ -0,0 +1,92 @@
+# Configure SSO with SAML using Okta
+
+## Overview
+
+Cloud Pak for Data can be configured to use external authenitcation through a SAML provider. If this is to be implemented, it should be implemented immediately after installation of the base and before the addition of any end-users to CPD. These steps are based on the [documentation for single sign-on](https://www.ibm.com/support/knowledgecenter/SSQNUZ_3.5.0/cpd/install/saml-sso.html).
```

Comment:
"authenitcation" misspelled

Reply:
fixed in https://github.com/ibm-hcbt/cp4d-install/pull/3/commits/81e270188e020b8dc1b641ee6017216a4f63c953